app/assets/javascripts/discourse/app/components/d-modal.gjs

```diff
@@ -228,7 +231,27 @@ export default class DModal extends Component {
     }
   }
 
+  #animateBackdropOpacity(position) {
+    const backdrop = this.wrapperElement.nextElementSibling;
+
+    if (!backdrop) {
```

Comment:
This is a little brittle, but 1) if we break this it's not extremely bad 2) this shouldn't change often